ontology/uco/observable/observable.ttl

```diff
@@ -6150,6 +6451,74 @@ observable:Software
 	sh:targetClass observable:Software ;
 	.
 
+observable:SoftwareBuild
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:Software ;
+	.
+
+observable:SoftwareBuildFacet
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf core:Facet ;
+	sh:property
+		[
+			sh:class observable:BuildUtility ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:buildUtility ;
+		] ,
+		[
+			sh:class observable:Compiler ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:compiler ;
+		] ,
+		[
+			sh:class observable:Library ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:library ;
+		] ,
+		[
+			sh:class observable:Package ;
+			sh:nodeKind sh:IRI ;
+			sh:path observable:package ;
+		] ,
+		[
+			sh:class observable:Script ;
+			sh:maxCount "1"^^xsd:integer ;
```

Comment:
Why is this maximum-count 1?  Some software follows a multi-step build process.  For instance, Autotools-style projects can still ostensibly build with `./bootstrap.sh && ./configure && make && make install`, which could challenge this max-1 constraint.  For another example, some IDEs provide a push-button build, not necessarily a script; would that build style induce `observable:buildScript` to be omitted?